blog/2025-01-14-build-3d-with-react.md

```diff
@@ -0,0 +1,33 @@
+---
+authors: mark
+slug: declarative-3d-with-playcanvas-react
+title: "Declarative 3D with React"
+tags:
+  - engine
+  - react
+---
+
+Since we introduced PlayCanvas, we've seen many developers around the world build amazing 3D experiences using the PlayCanvas Engine and Editor.
+
+Today we're proud to announce the release of **[Playcanvas React](https://github.com/playcanvas/react)**, a new declarative way to build 3D content using [React](https://react.dev/).
+
+[@playcanvas/react](https://github.com/playcanvas/react) is a thin react wrapper around PlayCanvas. It allows you to build rich 3D content in a declarative way that integrates with the React ecosystem. You can create rich, complex 3D content with assets, physics, gaussian splats, audio and more whilst leveraging the powerful state management of React.
```

Comment:
No need to link to the GH repo for every occurrence of 'PlayCanvas React' in the article.